src/services/programs.js

```diff
@@ -290,6 +293,18 @@ module.exports = class ProgramsHelper {
 				})
 			}
 
+			if (
+				fetchResource.status == common.RESOURCE_STATUS_PUBLISHED &&
```

Comment:
@adithyadinesh0412 If the review after publish is disabled ? do we need to copy the resource all the time ?